src/data/fintype/basic.lean

```diff
@@ -1116,6 +1117,15 @@ instance finset.fintype [fintype α] : fintype (finset α) :=
   [decidable_eq α] [decidable_eq β] : fintype (α ↪ β) :=
 fintype.of_equiv _ (equiv.subtype_injective_equiv_embedding α β)
 
+instance [decidable_eq α] [fintype α] {n : ℕ} : fintype (sym.sym' α n) :=
+begin
+  unfold sym.sym' vector.perm.is_setoid,
```

Comment:
Does it work if you remove this line?

Reply:
Nope :(
```
failed to synthesize type class instance for
α : Type u_1,
_inst_1 : decidable_eq α,
_inst_2 : fintype α,
n : ℕ
⊢ decidable_rel has_equiv.equiv
```
I figured out that this `has_equiv.equiv` was `λ (a b : vector α n), a.val ~ b.val`, but Lean doesn't seem to both unfold and look for the correct typeclass.